### PR TITLE
randomize tag id in quotation factory to fix RecordNotUnique

### PR DESCRIPTION
**Pull request by Henryvw**

### CODE REVIEW

File: spec/factories/quotations.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
   trait :tag do
     after(:build) do |quotation|
-      quotation.tags << create(:tag, name: 'Roman')
+      quotation.tags << create(:tag, name: "Roman#{random_id_generator}")
```
Comment (defsprite) [inline spec/factories/quotations.rb]: Why not make the `:name` attribute on the tag factory a sequence?

File: spec/models/quotation_spec.rb [hunk 2]
```diff
@@ -60,7 +60,8 @@
   describe 'tag-related instance methods' do
     context 'find quotations given tag(s)' do
       it '#tagged_with - when given a particular tag, returns all quotations attached to it in the database' do
-        expect(Quotation.tagged_with('Roman').first.title).to eq('Aurelius Dankenliste')
+        binding.pry
```
Comment (defsprite) [inline spec/models/quotation_spec.rb]: Please remove once finished before merging!

File: spec/factories/quotations.rb (hunk 1 above)
Comment (Henryvw) [reply spec/factories/quotations.rb]: Done.

Personally I must say that I find it a bit ugly/involved to combine the sequences with the traits:

```ruby
  sequence :roman_name do |n|
    "RomanName#{n}"
  end

  trait :tag do
    after(:build) do |quotation|
      quotation.tags << create(:tag, name: "#{generate :roman_name}")
    end
  end
```

But. Whatevs. It's a more proper way of doing it. I hear and agree with your general points you wrote elsewhere regarding the dangers of using my own `random_id_generator` as a custom sequencer.

File: spec/models/quotation_spec.rb (hunk 2 above)
Comment (Henryvw) [reply spec/models/quotation_spec.rb]: Pardon! Fixed

File: spec/factories/quotations.rb (hunk 1 above)
Comment (Henryvw) [reply spec/factories/quotations.rb]: Now updated further per our conversation yesterday.

**Comment by Henryvw** [comment]
> Where is the changed `tag` factory? (The one that uses the `tag_name_#{n}`-sequence?)
> 
> If it's on a different branch, but in a single commit this would be a nice use case for `git cherry-pick`

It's here on this branch! spec/factories/tags.rb. You can see the updated tag factory in this Pull Request

**Comment by Henryvw** [comment]
I just amended the commit - I replaced the last instance variables here with ` let() `

**Pull request merged**